src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedBackingFieldSymbol.cs

```diff
@@ -125,7 +128,7 @@ internal override void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttrib
         {
             base.PostDecodeWellKnownAttributes(boundAttributes, allAttributeSyntaxNodes, diagnostics, symbolPart, decodedData);
 
-            if (!allAttributeSyntaxNodes.IsEmpty && _property.IsAutoPropertyWithGetAccessor)
+            if (!allAttributeSyntaxNodes.IsEmpty && _property.BackingField is { IsCreatedForFieldKeyword: false })
```

Comment:
>BackingField [](http://example.com/codeflow?start=62&length=12)

This doesn't look like a change to the better. We traded one way to bind accessors with another.